boards/sparkfun/micromod/doc/index.rst

```diff
@@ -0,0 +1,212 @@
+##########################################
+
+Overview
+********
+
+The ``micromod/nrf52840`` board target features the nRF52840 SoC
+from Nordic Semiconductor, the SparkFun MicroMod nRF52840 Processor offers
+a powerful combination of ARM Cortex-M4 CPU and 2.4 GHz Bluetooth transceiver
+in the MicroMod form-factor. More information can be found in
+the `Micromod nRF52840 guide`_.
+
+Hardware
+********
+
+The ``micromod/nrf52840`` board target supports the following
+hardware features:
+
+- ARM Cortex-M4 CPU with floating point unit (FPU)
+   - 1MB internal Flash
+   - 256kB internal RAM
+- Integrated 2.4GHz radio with support for Bluetooth Low Energy (BLE) and ieee802154
+- USB 2.0 full speed (12 Mbps) controller
+- QSPI with 128Mb flash memory
+- ARM TrustZone Cryptocell 310 security subsystem
+- USB host mode compliant to 2.0 specification
+- GPIO
+- 2 UART
+- 2 I2C
+- 2 SPI
+- 2 ADC
+- 2 PWM
+- Watchdog Timer (WDT)
+
+Supported Features
+==================
+
+The Zephyr ``micromod/nrf52840`` board target supports the following hardware
+features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| SYSTICK   | on-chip    | system clock                        |
++-----------+------------+-------------------------------------+
+| RADIO     | on-chip    | Bluetooth,                          |
+|           |            | ieee802154                          |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port                         |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| PINMUX    | on-chip    | pinmux                              |
++-----------+------------+-------------------------------------+
+| FLASH     | on-chip    | flash                               |
++-----------+------------+-------------------------------------+
+| SPI       | on-chip    | spi                                 |
++-----------+------------+-------------------------------------+
+| I2C       | on-chip    | i2c                                 |
++-----------+------------+-------------------------------------+
+| USB       | on-chip    | usb                                 |
++-----------+------------+-------------------------------------+
+| QSPI      | on-chip    | nordic,nrf-qspi                     |
++-----------+------------+-------------------------------------+
+
+
+Connections and IOs
+===================
+
+LED
+---
+
+* Led0 (blue) = P0.13
+
+Micromod header
+---------------
+
+* micromod_1_uart = uart0
+* micromod_2_uart = uart1
```

Comment:
fwiw it's a bit confusing that this starts at 1 while i2c and spi start at 0

Reply:
I fully agree with you. However, it is the naming that Sparkfun uses in the micromod series (UART1, UART2, I2C(0) and I2C1: https://cdn.sparkfun.com/assets/learn_tutorials/1/2/0/6/SparkFun_MicroMod_Interface_v1.0_-_Pin_Descriptions.pdf

What do you think that is better?

Reply:
fine then :) I was mainly asking in case this was an oversight